frontend/src/workbench/Workbench.tsx

```diff
@@ -62,8 +62,14 @@ export const Workbench = ({
   const { selection, representations, displayedRepresentation } = context;
 
   const setSelection = (selection: Selection) => {
-    const isRepresentation = registry.isRepresentation(selection.kind);
-    const updateSelectionEvent: UpdateSelectionEvent = { type: 'UPDATE_SELECTION', selection, isRepresentation };
+    const representations: Representation[] = selection?.entries.filter((entry) =>
```

Comment:
no need for `?` the selection cannot be null here, can it?

Reply:
It should not. Removed.